docs/modules/ROOT/pages/cops_minitest.adoc

```diff
@@ -785,3 +785,34 @@ refute_respond_to(self, :do_something)
 === References
 
 * https://minitest.rubystyle.guide#refute-respond-to
+
+== Minitest/UnspecifiedException
+
+|===
+| Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
+
+| Pending
+| Yes
+| No
+| 0.10
+| -
+|===
+
+This cop checks for a specified error in `assert_raises`.
+
+=== Examples
+
+[source,ruby]
+----
+# bad
+assert_raises { do_something}
```

Comment:
Space after `do_something` was missing. Maybe we could rename `do_something` to `do_something_that_raises_exception`. Just more readable 😊 

```suggestion
assert_raises { do_something }
```

Reply:
Thanks! Changed it to `raise FooException`.